rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -143,6 +139,13 @@
 
     private static final String PATH_TASKS = "/tasks/";
 
+    private static final String IAM_LOGIN_METHOD = "IAMLoginMethod";
+
+    public static final Boolean IAM_IS_USED = PASchedulerProperties.SCHEDULER_LOGIN_METHOD.getValueAsString()
```

Comment:
I believe those constants appear several times. Can we factorize?

Reply:
Yes I agree, factorized